guides/admin/preconfigured-webhooks/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: single
+title:  "Preconfigured Webhook Stages"
+sidebar:
+  nav: guides
+---
+
+Preconfigured webhooks provide a simple, yet powerful, way of adding custom stages to Spinnaker. These stages are typically used to make quick API calls to an external system as part of a pipeline. Instead of extending the various components through code, users can simply add configuration to Orca for these stages. They appear in the UI as if they were a native stage.
+
+
+## Creating a Preconfigured Webhook Stage
+
+To create a preconfigured webhook stage, you'll need to add configuration for the stage in `orca-local.yml`. The `webhook.preconfigured` property supports configuring multiple webhook stages.
+
+```yaml
+webhook:
+  preconfigured:
+  - label: Custom Webook Stage
+    type: customWebhook
+    enabled: true
+    description: This describes the Custom Webhook Stage
+    method: GET
+    url: https://api.example.com
+```
+
+Preconfigured webhook stages support a variety of options, most of which are available via the UI when using the vanilla Webhook stage. Once a property is set within a preconfigured webhook stage, users will not be allowed to override that settings via the UI. The following basic properties can be set within your configuration. More advanced properties will be covered in other sections.
+
+* `enabled` - whether the stage will appear in the UI
+* `label` - the human readable name of the stage
+* `description` - a human readable description of what the stage is used for
+* `type` - a _unique_ key used to identifty the stage type within a pipeline
+* `url` - the url for the webhook
+* `customHeaders` - any headers needed for your webhook. ex. API tokens.
+* `method` - HTTP method used for the webhook.
+* `payload` - the JSON payload 
+
+## Preconfigured Webhook Properties
```

Comment:
The webhooks are preconfigured, per this feature, right? Not the properties? The wording of this heading sounds like the properties are preconfigured. 

Possible wording (and again, this assumes that I'm understanding the feature right): 
"Configure properties for custom webhooks"